.github/workflows/ci.yaml

```
@@ -306,16 +306,24 @@ jobs:
     if: needs.changes.outputs.security == 'true'
     uses: ./.github/workflows/grype_scan_reusable.yaml
+  # === Integration test pipeline (merge_group only) ===
 
-  integration-gate:
-    if: github.event_name == 'pull_request'
-    runs-on: ubuntu-latest
-    steps:
-      - run: echo "Integration tests run in the merge queue, not on PRs"
+  build_and_push:
+    if: github.event_name == 'merge_group'
+    uses: ./.github/workflows/reusable_build_and_push.yaml
```

actionlint (1.7.11)

[error] could not read reusable workflow file for "./.github/workflows/reusable_build_and_push.yaml": open /home/jailuser/git/.github/workflows/reusable_build_and_push.yaml: no such file or directory

(workflow-call)